x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authc/Authentication.java

```diff
@@ -141,6 +143,17 @@ public void writeTo(StreamOutput out) throws IOException {
         }
     }
 
+    public static void writeTo(Authentication authentication, StreamOutput out) throws IOException {
+        User.writeTo(authentication.getUser(), out);
+        authentication.getAuthenticatedBy().writeTo(out);
+        if (authentication.getLookedUpBy() != null) {
+            out.writeBoolean(true);
+            authentication.getLookedUpBy().writeTo(out);
+        } else {
+            out.writeBoolean(false);
+        }
+    }
+
```

Comment:
why not use the `public void writeTo(StreamOutput out) throws IOException` above ?

Reply:
possibly a leftover from a previous attempt (?) - fixed.